app/src/components/widgets/charts/cumulativeTrendChart/chartjsConfig.js

```diff
@@ -64,131 +43,153 @@ export const generateChartColors = (widget) => {
   return colors;
 };
 /* eslint func-names: ["error", "never" ] */
-const DataSet = function(label, group) {
-  this.label = label;
+const DataSet = function(field, group, options) {
```

Comment:
What is a usual function in your mind? Is it arrow function? We cannot use them because we need to preserve context "this".

Reply:
You use it as a constructor later (`new DataSet(...)`). Is it necessary? Why not a function which returns an object?

Reply:
We can create new object using few different approaches, the keyword new+constructor is one of them. I can refactor this part to use literal notation `{}`

Reply:
If you need only an object, it's better to use object notation instead of constructor. Otherwise, use `class` keyword.
In this case I don't see necessity of using class.

Reply:
fixed